enterprise/models.py

```diff
@@ -197,6 +197,24 @@ class Meta:
         )
     )
 
+    TYPE_ENTERPRISE = 'enterprise'
+    TYPE_PARTNER = 'partner'
+    TYPE_NON_PROD = 'non_prod'
```

Comment:
A suggestion for better readability: 
```
Class CustomerType:
          """"Possible types of customers """
          ENTERPRISE = 'enterprise'
          PARTNER = 'partner'
          NON_PROD = 'non_prod'

```
You can refer anywhere in code something like that --> CustomerType.ENTERPRISE , CustomerType.PARTNER ...

Reply:
I have add new choices according to current implementation and what django docs suggest: https://docs.djangoproject.com/en/2.0/ref/models/fields/#choices. Also we still should be able to access the choices from `EnterpriseCustomer` model, e.g. `EnterpriseCustomer.TYPE_ENTERPRISE`.

If we want to define the choices as a class then we have to use Enum which is not needed for our current simple usecase in my opinion:
* http://anthonyfox.io/2017/02/choices-for-choices-in-django-charfields/
* https://hackernoon.com/using-enum-as-model-field-choice-in-django-92d8b97aaa63

Reply:
I suggest, using a Enum or Class would be a good approach instead of adding a pre-word `TYPE` for every choices.

Reply:
We have achieved this through introducing a new model `EnterpriseCustomerType`.